src/main.rs

```diff
@@ -289,7 +289,7 @@ fn create_app<'a, 'b>(
                 .help(
                     "vhost-user-block backend parameters \"image=<image_path>,\
                      sock=<socket_path>,num_queues=<number_of_queues>,\
-                     readonly=true|false,direct=true|false\"",
+                     readonly=true|false,direct=true|false,poll_queue=true|false\"",
```

Comment:
Could you please update the OpenAPI definition here: https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/vmm/src/api/openapi/cloud-hypervisor.yaml

Reply:
Sure, I'll do that in a separate patch.

Reply:
Thanks :)

Reply:
Actually @rbradford pointed me that you don't need to do anything since you're not modifying any of the VMM's device. Sorry I got confused because now the `main.rs` CLI includes options for `vhost-user` backends.

Reply:
I was about to ask about this. Should I add `poll_queue` to `DiskConfig` and CH's `disk` argument? Reading the code, I looks to me that `block-backend` and `disk` need to be synchronized.

Reply:
@slp See #787 and #786